src/dl_reflect.cpp

```diff
@@ -122,12 +123,27 @@ dl_error_t DL_DLL_EXPORT dl_reflect_get_type_members( dl_ctx_t dl_ctx, dl_typeid
 	if(members_size < type->member_count)
 		return DL_ERROR_BUFFER_TOO_SMALL;
 
+	if (!dl_ctx->default_data_patched)
```

Comment:
This is the internal state I want to avoid by storing the default data in a persistant memory location instead of reallocating a buffer and moving around the data every time new types are added/loaded, discuss it here https://github.com/wc-duck/datalibrary/issues/161#issuecomment-1295106957